trade-finance-manager-ui/server/controllers/case/cancellation/validation/validate-effective-from-date.test.ts

```diff
@@ -0,0 +1,148 @@
+import { applyStandardValidationAndParseDateInput } from '@ukef/dtfs2-common';
+import { add, startOfDay } from 'date-fns';
+import { getErrorObjectFromMessageAndRefs, validateEffectiveFromDate } from './validate-effective-from-date';
+import { EffectiveFromDateValidationViewModel } from '../../../../types/view-models';
+
+jest.mock('@ukef/dtfs2-common', () => ({ applyStandardValidationAndParseDateInput: jest.fn() }));
+
+describe('validate effective from date', () => {
+  describe('getErrorObjectFromMessageAndRefs', () => {
+    it('correctly formats error messages', () => {
+      const testErrorMessage = 'test error message';
+      const testRefs = ['ref1', 'ref2'];
+
+      const result = getErrorObjectFromMessageAndRefs(testErrorMessage, testRefs);
+
+      const expectedResult = {
+        errors: {
+          summary: [{ text: testErrorMessage, href: '#ref1' }],
+          effectiveFromDateError: {
+            message: testErrorMessage,
+            fields: testRefs,
+          },
+        },
+      };
+
+      expect(result).toEqual(expectedResult);
```

Comment:
Should be `expected` instead of `expectedResult`, not a blocker - need to do this in a few places, i'll raise a PR 👍